src/utils/error.ts

```diff
@@ -649,6 +649,10 @@ export class AuthClientErrorCode {
     code: 'not-found',
     message: 'The requested resource was not found.',
   };
+  public static USER_NOT_DISABLED = {
+    code: 'user-not-disabled',
+    message: 'The user must be disabled in order to bulk delete it (or you must pass force=true).',
```

Comment:
Didn't we decide against the `force=true` option in this API?

Reply:
I am confused about this. This was approved in https://docs.google.com/document/d/1mG0ilmh8H-C6DYXA9oOQUro2MP0on_9k7Qf4oCLRyBk/edit#